apps/web/src/styles/globals.css

```
@@ -66,11 +66,11 @@
     --muted: #181818;
     --muted-foreground: #999999;
 
-    --table-foreground: #999999;
-    --table-background: #181818;
-    --table-bakcground-alt: #424141;
-    --table-header-background: #84888b;
-    --table-header-foreground: #0d0d0e;
+    --table-foreground: #f3f0f0;
+    --table-background: #151210;
+    --table-bakcground-alt: #1f1a17;
```

Fant buggen som gjorde bakgrunnen lys 
```suggestion
    --table-background-alt: #1f1a17;
```